bot/exts/info/codeblock/_parsing.py

```diff
@@ -12,7 +12,7 @@
 log = logging.getLogger(__name__)
 
 BACKTICK = "`"
-PY_LANG_CODES = ("python", "pycon", "py")  # Order is important; "py" is last cause it's a subset.
+PY_LANG_CODES = ("python", "python-repl", "pycon", "py")  # Order is important; "py" is last cause it's a subset.
```

Comment:
Seeing the comment at the end of the line, `python-repl` should be before `python`. Have you tested the new regex?

Reply:
I interpreted the bit about "subset" as referring uniquely to `"py"` -- being a subset of the other/prior elements. Reinserting `python-repl` at the beginning of the tuple, however, does not escape the regex, either.